### PR TITLE
To do list Code review

### PR DESCRIPTION
**Pull request by ngeelani48**
**Kindly Review this code and give your valuable comments.**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+FnToDoList(Tasks);
+
+input.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter' && input.value !== '') {
+    addTask(Tasks, input.value);
+    input.value = '';
+    updateLocalStorage(Tasks);
+    FnToDoList(Tasks);
+  }
+});
```
Comment (SaneMyburg) [inline src/index.js]: [] Kindly use proper descriptive names for naming your variables and functions, this is good practice for ease of readability.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+List.addEventListener('click', (e) => {
+  if (e.target.closest('.list-li-checkbox')) {
+    markTask(e, Tasks);
+  }
+  updateLocalStorage(Tasks);
+  FnToDoList(Tasks);
+});
+
```
Comment (SaneMyburg) [inline src/index.js]: [] Instead of using 'e', please use proper naming like event, to make it easy to read and your code professional.

**Review by SaneMyburg — CHANGES_REQUESTED** [review]
Well done coming this far 👏👏
However, there are a few changes, which I have highlighted below.

## Highlights
 Profesional README ✔️
 All linter checks passing ✔️
 No merge conflicts ✔️
 Correct GitHub flow ✔️

File: src/index.js (hunk 1 above)
Comment (ngeelani48) [reply src/index.js]: @SaneMyburg Thanks for your suggestion. 🤝

File: src/index.js (hunk 2 above)
Comment (ngeelani48) [reply src/index.js]: @SaneMyburg Thank You for your time and valuable suggestions.

**Review by SaneMyburg — APPROVED** [review]

**Review by SaneMyburg — APPROVED** [review]
Well done, the project is approved you can now merge your PR.

**Pull request merged**